t/25-cache-service.t

```diff
@@ -144,16 +144,11 @@ sub test_download {
 
     ok $cache_client->enqueue($asset_request), 'enqueued';
 
-    my $state = $cache_client->status($asset_request);
-    $state = $cache_client->status($asset_request) until ($state ne OpenQA::CacheService::Model::Cache::STATUS_IGNORE);
-
-    # After IGNORE, only DOWNLOAD status could follow, but it could be fast enough to not catch it
-    $state = $cache_client->status($asset_request);
-    $state = $cache_client->status($asset_request)
-      until ($state ne OpenQA::CacheService::Model::Cache::STATUS_DOWNLOADING);
+    my $status = $cache_client->status($asset_request);
+    $status = $cache_client->status($asset_request) until !$status->is_downloading;
```

Comment:
Not even a small sleep between the polls?

Reply:
Yea, that surprised me too. The test jobs are very fast though, so it probably doesn't really matter.